src/main/java/com/springml/salesforce/wave/api/ForceAPI.java

```diff
@@ -47,4 +48,12 @@
 //    public InsertObjectsResponse insertObjects(String object, List<String> objects) throws Exception;
 
     public String getSFEndpoint() throws Exception;
+
+    /**
+     * Returns list of the salesforce object column names
+     * @param object Name of the salesforce object
+     * @return
+     * @throws Exception
+     */
+    public DescribeSObjectResult describeSalesforceObject(String object) throws Exception;
```

Comment:
similar compatibility issue here, we need to confirm spark-salesforce is the only downstream app. @samuel-pt

Reply:
@mengxr  Currently spark-salesforce is the only downstream app. So no issues